tests/mock_tests/schedules/test_schedule_mock.py

```diff
@@ -41,7 +41,7 @@ def tearDownClass(cls) -> None:
 
     def test_get_schedule(self, m ):
         """get_schedule should return schedules for date, starDate, endDate"""
-        m.get('https://statsapi.mlb.com/api/v1/schedule?sportId=1&startDate=04/07/2022&endDate=04/07/2022', json=self.mock_schedule,
+        m.get('https://statsapi.mlb.com/api/v1/schedule?date=04%2F07%2F2022&sportId=1', json=self.mock_schedule,
```

Comment:
Does this still work with the special characters?

Reply:
Yup it does, but let me go and change that to slashes